examples/demo/src/index.ts

```diff
@@ -1,14 +1,9 @@
-import './index.css';
 import { parse } from 'jsonc-parser';
 import * as monaco from 'monaco-editor';
 import { configureMonacoTailwindcss, tailwindcssData } from 'monaco-tailwindcss';
 import { Config } from 'tailwindcss';
 
-declare global {
```

Comment:
hi ;)
just wondering why we dont nerd this anymore ...

Reply:
This was fixed upstream in https://github.com/microsoft/monaco-editor/pull/3137